api/api.proto

```diff
@@ -514,8 +514,11 @@ message VolumeSpec {
   Xattr.Value xattr = 36;
   // ScanPolicy specifies the filesystem check policy
   ScanPolicy scan_policy = 37;
-  // MountOptions defines the options that should be used while mounting this volume
+  // MountOptions defines the options that will be used while mounting this volume
   MountOptions mount_options = 38;
+  // Sharedv4ClientMountOptions defines the options that will be used while mounting a sharedv4 volume
+  // from a node where the volume replica does not exist
+  MountOptions sharedv4_client_mount_options = 39;
```

Comment:
A little shorter sharedv_mount_options?

Reply:
I wanted to distinguish between MountOptions and Sharedv4MountOptions. The other MountOptions field is applicable to sharedv4 as well.